pkgs/games/xash3d/wrapper.nix

```diff
@@ -0,0 +1,59 @@
+{ writeShellScriptBin
+, makeDesktopItem
+, symlinkJoin
+, hlsdk
+, xash3d-unwrapped
+, name # codename, normally matches gamedir
+, fullname # the human readable name (e.g. "Half-Life: Opposing Force")
+, gamedir ? name # the name of the directory containing game data (e.g. "gearbox")
+, basedir ? "$HOME/.local/share/xash3d"
```

Comment:
Should probably handle `XDG_DATA_HOME`, see https://github.com/vs49688/nix-repo/blob/8987b66fe2bb61a77955986826b9cd0c37f2b96c/pkgs/xash3d-fwgs/launch.sh#L21C1-L31C1

```bash
if [[ -z "$XDG_DATA_HOME" ]]; then
    export XDG_DATA_HOME="$HOME/.local/share"
fi

if [[ -z "$XASH3D_BASEDIR" ]]; then
    export XASH3D_BASEDIR="$XDG_DATA_HOME/xash3d-fwgs/"
fi
```

Reply:
Yes I had thought about setting it with nix but this is probably better